scripts/contextMenu.js

```diff
@@ -0,0 +1,82 @@
+const ContextMenu = (() => {
```

Comment:
Using module pattern here, to create scope when this script is inserted in the browser.